js/ui/overlay/ui.overlay.js

```diff
@@ -432,8 +435,18 @@ const Overlay = Widget.inherit({
     },
 
     _renderWrapperAttributes() {
-        const { wrapperAttr } = this.option();
-        this._$wrapper.attr(wrapperAttr ?? {});
+        const { wrapperAttr } = this.option() || {};
+        const attributes = extend({}, wrapperAttr);
+        const classNames = attributes.class;
+
+        delete attributes.class;
+
+        this.$wrapper()
+            .attr(attributes)
+            .removeClass(this._customWrapperClass)
+            .addClass(classNames);
+
+        this._customWrapperClass = classNames;
```

Comment:
can we get rid of this cache? For example, pass `wrapperAttr` previous value from option changed handler

Reply:
decided to stay with cache